app/pages/project/index.cjsx

```diff
@@ -464,8 +474,13 @@ ProjectPageController = React.createClass
 
   render: ->
     slug = @props.params.owner + '/' + @props.params.name
+    betaApproved = @state.project?.beta_approved
 
     <div className="project-page-wrapper">
+
+      {if betaApproved
+        <div className="beta-border"></div>}
```

Comment:
The `.beta-border` class doesn't play well with the outer `div` and restricts scrolling, so I added it back as a separate `div`.